package-examples/kustomize-pkg/README.md

```diff
@@ -0,0 +1,158 @@
+# kustomize-pkg
+
+This is a simple package that shows how kpt packages can be used instead of remote bases in kustomize.  That allows you to take advantage of kpt's rebase and local, in-place edits.
+
+## Steps
+
+1. [Fetch the package](#fetch-the-package)
+2. [View the package contents](#view-the-package-contents)
+3. [kustomize the config](#kustomize-the-config)
+4. [Render the kustomization](#render-the-kustomization)
+5. [Apply the package](#apply-the-package)
+6. [Clean up resources](#clean-up-resources)
+
+### Fetch the package
+
+Get the example package on to local using `kpt pkg get`. Note that this package is for this example, wihin that package we are also using the nginx sub-package which is an alternative to having a remote base. 
+
+```shell
+$ kpt pkg get https://github.com/GoogleContainerTools/kpt.git/package-examples/kustomize-pkg
```

Comment:
don't you want to specify the `next` branch?
Does these commands work currently?

Reply:
this is a copy from the template and the rest of them are using the default branch which will be main soon.  I don't think we should single this sample out.

Reply:
are you saying no one should try this example before we make `next` default?

Reply:
We have all the rest of the samples that do not point to a branch.  I am fine making this point to next just to move forward but until now no one has complained about the other samples. 

Two theories:
1) people looking at samples already understand how to point at the right branches and are in tune with the product lifecycle
2) people are looking at the samples to just understand the flow and read the pattern.

Reply:
I will change it to next, please update it to point to default branch when you change the branches over.

Reply:
Two wrongs don't make a right. People are consuming wordpress and nginx through the book which have correct versioning using tags. Just add the next branch to make this work in the meantime.

Reply:
all the existing samples are updated.